PYME/DSView/dsviewer.py

```diff
@@ -142,7 +142,7 @@ def __init__(self, image,  parent=None, title='', mode='LM',
         #a submenu for modules to hook and install saving functions into
         self.save_menu = wx.Menu()
         self._menus['Save'] = self.save_menu
-        tmp_menu.AppendMenu(-1, 'Save &Results', self.save_menu)
+        tmp_menu.Append(-1, 'Save &Results', self.save_menu)
```

Comment:
We should probably change these to use the AUIFrame methods ...

Reply:
Sure